libs/sdk-ui-semantic-search/src/ResultsItem.tsx

```diff
@@ -27,52 +29,84 @@ export type ResultsItemProps = {
      */
-    onHover: () => void;
+    onHover: (item: ISemanticSearchResultItemWithUrl) => void;
     /**
      * Callback to trigger when the item is clicked.
      */
-    onClick: () => void;
+    onSelect: (item: ISemanticSearchResultItemWithUrl) => void;
 };
 
 /**
  * A single result item in the search results.
  * @internal
  */
-export const ResultsItem: React.FC<ResultsItemProps> = ({ item, onHover, onClick, selected }) => {
-    const onMouseEnter = (e: React.MouseEvent<HTMLDivElement>) => {
+export const ResultsItem: React.FC<ResultsItemProps> = ({ item, onHover, onSelect, active }) => {
+    const theme = useTheme();
+    const onMouseEnter = (e: React.MouseEvent<HTMLAnchorElement>) => {
         // Prevent multiple triggers
         if (e.target !== e.currentTarget) {
             return;
         }
 
-        onHover();
+        onHover(item);
+    };
+
+    const onClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+        e.preventDefault();
+        onSelect(item);
     };
 
     const className = classnames({
         "gd-semantic-search__results-item": true,
-        "gd-semantic-search__results-item--selected": selected,
+        "gd-semantic-search__results-item--active": active,
     });
 
     // Use mouse enter with the target check instead of hover to prevent multiple triggers
-    // Use mouse down instead of click to prevent the dialog from disappearing before the click event is caught
     return (
-        <div className={className} onMouseEnter={onMouseEnter} onMouseDown={onClick}>
-            <div className="gd-semantic-search__results-item__icon">{renderItemIcon(item)}</div>
-            <div className="gd-semantic-search__results-item__text">
-                <Typography tagName="p">{item.title}</Typography>
-            </div>
-        </div>
+        <a className={className} onMouseEnter={onMouseEnter} onClick={onClick} href={item.url}>
+            <span className="gd-semantic-search__results-item__icon">{renderItemIcon(item)}</span>
+            <span className="gd-semantic-search__results-item__text">{item.title}</span>
+            <span className="gd-semantic-search__results-item__details">
+                <BubbleHoverTrigger eventsOnBubble>
+                    <Icon.QuestionMark color={theme?.palette?.complementary?.c7 ?? "#6D7680"} />
+                    <Bubble
+                        className="bubble-light gd-semantic-search__bubble"
+                        alignPoints={BUBBLE_ALIGN_POINTS}
+                        arrowStyle={{ display: "none" }}
+                    >
+                        {/* It's OK to have div inline, as this chunk is rendered through portal */}
+                        <div className="gd-semantic-search__results-item__details__contents">
+                            <Typography tagName="h3">{item.title}</Typography>
+                            {renderIfHasDescription(item)}
+                            <h4>ID</h4>
```

Comment:
Missing localization

Reply:
Do we want localization for ID?

Reply:
We are localizing all the texts. I would not pick them based on some assumption as we don't know how ID is written in Chinese for example :)